src/components/Carousel.js

```diff
@@ -105,6 +106,10 @@ class Carousel extends Component {
     this.onResize();
     // setting autoplay interval
     this.resetInterval();
+
+    if (typeof this.props.handleIsReady !== 'undefined') {
+      this.props.handleIsReady();
+    }
```

Comment:
```suggestion
    if (typeof this.props.handleIsReady === 'function') {
      this.props.handleIsReady();
    }
```

If it has to be function perhaps it's a better idea to check type explicitly. WDY?